src/Bicep.Core.Samples/Files/Completions/booleanValues.json

```diff
@@ -6,8 +6,11 @@
     "deprecated": false,
     "preselect": true,
     "sortText": "1_false",
-    "insertText": "false",
-    "insertTextFormat": "plainText"
+    "insertTextFormat": "plainText",
+    "textEdit": {
+      "range": {},
```

Comment:
The `range` on every completion is different for any position where completions are triggered. If I left it in, we'd have an explosion of assert files.